Dockerfile

```diff
@@ -93,8 +69,9 @@ ARG PULUMI_VERSION
 RUN curl -fsSL https://get.pulumi.com/ | bash -s -- --version $PULUMI_VERSION && \
   mv ~/.pulumi/bin/* /usr/bin
 
-RUN pulumi plugin install resource gcp
-RUN pulumi plugin install resource random
+RUN curl -fsSLo /tmp/dch.tgz https://github.com/docker/docker-credential-helpers/releases/download/${DOCKER_PASS_CH}/docker-credential-pass-${DOCKER_PASS_CH}-amd64.tar.gz && \
+    pulumi plugin install resource docker-credential-helpers ${DOCKER_PASS_CH} -f /tmp/dch.tgz && \
+    rm -f /tmp/dch.tgz
```

Comment:
can this be installed this way as a credential helper?

Reply:
yeah, I'll revert that.. good catch